README.md

```diff
@@ -11,19 +11,32 @@ To submit a bug or suggest a documentation fix/enhancement for a specific page,
 ## Edit the site
 To edit a single page, navigate to the page, and click "Edit this page on GitHub" in the page footer. That will create a simple Pull Request where you can propose your changes.
 
-You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that. See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
+You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
 
 See [MAINTAINING.md](MAINTAINING.md) for information about site maintainers.
 
 See [README_ORGANIZATION.md](README_ORGANIZATION.md) for information about how the source code is organized.
 
-All content in the website is written in Markdown. For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).
+Most of the content in the website is written in Markdown.
+For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).
 
 You can create a new file two ways:
 
 - From your local Git repo folder, navigate to the relevant directory, create a new text file and save it with the following naming convention: `<name>.md`
 
 - From the GitHub repo, navigate to the relevant directory and click the New File button at the top-right corner of the pane.
 
+After you submit a PR to the ovirt-site reposiory, automation will verify your submission. If no error is found you'll see a green symbol and a text saying
+"All checks have passed". You can inspect the generated website by downloading the site-without-markdown.zip file from the automation system.
+In order to do so, click on the "Checks" tab within your PR and then click on "Website Tests" link on the left bar.
+You can find "site-without-markdown" artifact at the bottom of the page.
+
+Once downladed the zip file, in order to check how the website looks like you need to extract the archive and serve the extracted directory with a web server.
+An easy way to do it is using python: `python3 -m http.server` from the web site directory. This won't provide a full replica of what will happen on the real website

```

Comment:
I was able to run the server and load the site home page, but I couldn't open any documentation, and there was no styling on the site. When I clicked on Documentation, I got:
![image](https://user-images.githubusercontent.com/39279277/119495444-1831cf80-bd6b-11eb-854d-52c2e8874215.png)